plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php

```diff
@@ -276,6 +289,36 @@ private static function order_received( $order_id = 0 ) {
 		// Empty current cart.
 		wc_empty_cart();
 
+		// If the specified order ID was invalid, we still render the default order received page (which will simply
+		// state that the order was received, but will not output any other details: this makes it harder to probe for
+		// valid order IDs than if we state that the order ID was not recognized).
+		if ( ! $order ) {
+			wc_get_template( 'checkout/thankyou.php', array( 'order' => false ) );
+			return;
+		}
+
+		$order_customer_id = $order->get_customer_id();
+
+		// For non-guest orders, require the user to be logged in before showing this page.
```

Comment:
@jonathansadowski Is there a ticket or some documentation/discussion about this change? The update from 7.8.0 to 7.8.1 broke our guest checkout and I'd like to find out if it's an issue on our side or a woocommerce issue.